src/drone_controller/calculation_layer/thrust_calculator.py

```diff
@@ -0,0 +1,20 @@
+# pylint: disable=too-few-public-methods
+class ThrustCalculator:
+    """
+    Calculator for rotor thrusts.
+    """
+
+    def __init__(self, drone_state, expected_state):
+        """
+        :param drone_state: the momentary drone state
+        :param expected_state: the expected state of the dron
+        """
+        self.drone_state = drone_state
+        self.expected_state = expected_state
+
+    def calc(self, rotor_count):
```

Comment:
I doubt, that you can use the same calculation for different number of rotors.
Physic is usually not that simple :P